scripts/checks.sh

```diff
@@ -44,8 +44,8 @@ TIMEOUT_CMD="timeout --kill-after=$(( TIMEOUT * 2 )) ${TIMEOUT}"
 mountpoint="/var/lib/${ENG}"
 
 external_fqdn="0.resinio.pool.ntp.org"
-IPV4_ADDRESS="1.1.1.1"
-IPV6_ADDRESS="2606:4700:4700:6400"
+IPV4_ADDRESS="1.1.1.1" # https://1.1.1.1/dns/
+IPV6_ADDRESS="2606:4700:4700::6400" # https://developers.cloudflare.com/1.1.1.1/ipv6-networks
```

Comment:
Note the extra colon character before 6400.

On a device that has an IPv6 stack:

Before this PR:
```
root@a459e16:~# ip -6 route get "2606:4700:4700:6400"; echo $?
ip: need at least destination address
1
```
After this PR:
```
root@a459e16:~# ip -6 route get "2606:4700:4700::6400"; echo $?
2606:4700:4700::6400 from :: via 2001:470:1f1c:1dd::1 dev he-6in4  src 2001:470:1f1c:1dd::2  metric 675
0
```

On a device that does **not** have an IPv6 stack:

Before this PR:
```
root@2eb94bd:~# ip -6 route get "2606:4700:4700:6400"; echo $?
ip: need at least destination address
1
```
After this PR:
```
root@2eb94bd:~# ip -6 route get "2606:4700:4700::6400"; echo $?
ip: RTNETLINK answers: Network is unreachable
2
```